### PR TITLE
comparing QC methods betweeen nama and khomani data

### PR DESCRIPTION
**Pull request by dralhindi**

### CODE REVIEW

File: Pheno/scripts/QC_script.R [hunk 1]
```diff
@@ -22,41 +27,31 @@ data$avg.m.r <- NA
...
-       "m.r.arm_1", "m.r.arm_2", "m.r.arm_3", "m.r.arm_4", "m.r.arm_5",
-       "m.l.arm_1", "m.l.arm_2", "m.l.arm_3", "m.l.arm_4", "m.l.arm_5")
-
-data[, cols] <- sapply(data[, cols], as.numeric)
+data[, c(5:24)] <- sapply(data[, c(5:24)], as.numeric)
+colnames(data)
 
 ## Loop through rows of data frame "data"
 ## Drops min & max values, averages them, puts avg in another column in data frame
+## Does this for each measurement separately: e_r, e_l, m_r, m_l
```
Comment (najisawas) [inline Pheno/scripts/QC_script.R]: *Do this for each...

File: Pheno/scripts/QC_script.R [hunk 2]
```diff
@@ -67,5 +62,7 @@ for (row in seq_len(nrow(data))) {
   data[row, ]$Final_M <- avg_M
 }
 
+nrow(data)
```
Comment (najisawas) [inline Pheno/scripts/QC_script.R]: i dont think this is needed?

File: Pheno/scripts/khoesan_pigmentation_histogram_script.R [hunk 3]
```diff
@@ -0,0 +1,76 @@
+library(ggplot2)
+library(dplyr)
+library(hrbrthemes)
+library(viridis)
+library(RColorBrewer)
+
+setwd("/Users/dana.alhindi/DDRIG/Pheno/data_private")
```
Comment (najisawas) [inline Pheno/scripts/khoesan_pigmentation_histogram_script.R]: delete

File: Pheno/scripts/khoesan_pigmentation_histogram_script.R [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+# Plotting histograms for SA pigmentation data from 4 populations using ggplot2
+
+# Load data
+master <- read.csv("MasterPheno_All.csv")
```
Comment (najisawas) [inline Pheno/scripts/khoesan_pigmentation_histogram_script.R]: add `DDRIG/Pheno/data_private` here

File: Pheno/scripts/khoesan_pigmentation_histogram_script.R [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+  labs(title = "Melanin Distribution in KhoeSan Populations", y = "Melanin Content",
+       caption = "Source: Henn Lab pigmentation data") +
+  scale_color_brewer(palette = "Dark2") +
+  theme(legend.position = "none")
+
+
+
+
+# Save File
+pdf(file = "/Users/dana.alhindi/DDRIG/Pheno/output/distribution_pigmentation_plot.pdf", width = 12, height = 9)
```
Comment (najisawas) [inline Pheno/scripts/khoesan_pigmentation_histogram_script.R]: remove /Users/dana.alhindi/DDRIG

File: Pheno/scripts/QC_script.R (hunk 2 above)
Comment (dralhindi) [reply Pheno/scripts/QC_script.R]: I wanted it for visual to compare how many individuals were removed in the process of QC

Comment (najisawas) [reply Pheno/scripts/QC_script.R]: i see. I dont think it's adding to the actual functionality of the file however. If you ever need to visualize something like this you can simply run `nrow(data)` in your RStudio console. But here it's simply bloat

Comment (dralhindi) [reply Pheno/scripts/QC_script.R]: ok... ur right

**Review by najisawas — APPROVED** [review]
🎉 🎉

**Pull request merged**